src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +16,14 @@
      * "Can't get min value from list: method_input_list"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(s -> s.split(","))
+                .flatMap(a -> Arrays.stream(a))
+                .mapToInt(s -> Integer.parseInt(s))
+                .filter(i -> i % 2 == 0)
+                .min()
+                .orElseThrow(() -> new RuntimeException("Can't get min value from list:"
+                        + " method_input_list"));
```

Comment:
```suggestion
                        + numbers));
```
this way it will be more informative